packages/react-core/src/components/Progress/Progress.tsx

```diff
@@ -40,7 +40,7 @@ export class Progress extends React.Component<ProgressProps> {
   static defaultProps: ProgressProps = {
     className: '',
     measureLocation: ProgressMeasureLocation.top,
-    variant: ProgressVariant.info,
+    variant: null,
```

Comment:
so by default it will not set a variant class right

Reply:
Yep! `ProgressVariant.info` didn't exist and was doing nothing.